jobs/postfix/templates/opendkim/etc/opendkim/domainkeys/cloudgov.private.erb

```diff
@@ -0,0 +1 @@
+<% if_p('postfix.dkim_privatekey') do %><%= p('postfix.dkim_privatekey') %><% end %>
```

Comment:
I get that we're probably stuck maintaining this for a while, but `cloudgov.private.erb`?  Probably should rethink this filename so this package is re-usable.

Reply:
Good point.  Will fix the name.